governance/xc-admin/packages/xc-admin-frontend/hooks/usePyth.ts

```diff
@@ -193,7 +196,9 @@ const usePyth = (): PythHookData => {
       }
     })()
 
-    return () => {}
+    return () => {
+      cancelled = true
```

Comment:
without this line, it broke the site because it was returning multiple cluster data and the order wasn't preserved

Reply:
Me not understand but I'll trust you